airflow/utils/db.py

```diff
@@ -508,6 +508,22 @@ def create_default_connections(session: Session = NEW_SESSION):
         ),
         session,
     )
+    merge_conn(
+        Connection(
+            conn_id='redshift_default',
+            conn_type='redshift',
+            extra="""{
+    "iam": true,
+    "cluster_identifier": "my-redshift-cluster-identifier",
+    "port": 5439,
+    "profile": "default",
+    "db_user": "awsuser",
+    "database": "dev",
+    "region": "us-east-1"
```

Comment:
Is it ok to specify the region as the default parameter?

Reply:
It's a good question. It was to give an example but it also can be confusing and make the user think only us-east-1 is available, let me fix that

Reply:
Good point. I think thoguh leavin it empty is a better Idea ? What happens if someone uses the `<>` one?